extras/DoomBuilder configurations/Edge_Classic.cfg

```diff
@@ -0,0 +1,234 @@
+/*************************************************************\
+	Doom Builder Game Configuration for Edge Classic
+	(https://github.com/edge-classic/EDGE-classic)
+\*************************************************************/
+
+// This is required to prevent accidental use of a different configuration
+type = "Doom Builder 2 Game Configuration";
+
+// This is the title to show for this game
+game = "EDGE Classic";
+
+// This is the simplified game engine/sourceport name
+engine = "edgeclassic";
+
+// The format interface handles the map data format
+formatinterface = "DoomMapSetIO";
+
+// Default lump name for new map
+defaultlumpname = "MAP01";
+
+// Default testing parameters
+testparameters = "-iwad \"%WP\" -skill \"%S\" -file \"%AP\" \"%F\" -warp %L1 %L2 %NM";
+
+// Default nodebuilder configurations
+defaultsavecompiler = "glbsp_normal";
+defaulttestcompiler = "glbsp_fast";
+
+// Skill levels
+skills
+{
+	include("Includes\\Doom_misc.cfg", "skills");
+}
+
+// When this is set to true, sectors with the same tag will light up when a line is highlighted
+linetagindicatesectors = true;
+
+// Special linedefs
+soundlinedefflag = 64;	// See linedefflags
+singlesidedflag = 1;	// See linedefflags
+doublesidedflag = 4;	// See linedefflags
+impassableflag = 1;
+upperunpeggedflag = 8;
+lowerunpeggedflag = 16;
+
+// Door making
+makedoortrack = "DOORTRAK";
+makedooraction = 1;			// See linedeftypes
+
+// Generalized actions
+generalizedlinedefs = true;
+generalizedsectors = true;
+
+// Texture loading options
+mixtexturesflats = false;
```

Comment:
I'm not too familiar with DB2, but if this refers to being able to use flats as textures and vice versa, EC is capable of doing so